src/operators/ops_regular_cartesian_grid.jl

```diff
@@ -28,8 +27,23 @@ using Oceananigans:
 @inline avgz_f2c(g::RegularCartesianGrid{T}, f, i, j, k) where T = @inbounds T(0.5) * (f[i, j, k+1] + f[i, j, k])
 @inline avgz_f2e(g::RegularCartesianGrid{T}, f, i, j, k) where T = @inbounds T(0.5) * (f[i, j, k] + f[i, j, k-1])
 
-@inline fv(g::RegularCartesianGrid{T}, v, f, i, j, k) where T = T(0.5) * f * (avgy_f2c(g, v, i-1,  j, k) + avgy_f2c(g, v, i, j, k))
-@inline fu(g::RegularCartesianGrid{T}, u, f, i, j, k) where T = T(0.5) * f * (avgx_f2c(g, u, i,  j-1, k) + avgx_f2c(g, u, i, j, k))
+@inline ∂x_p(i, j, k, grid::RegularCartesianGrid, p) = δx_c2f(grid, p, i, j, k) / grid.Δx
+@inline ∂y_p(i, j, k, grid::RegularCartesianGrid, p) = δy_c2f(grid, p, i, j, k) / grid.Δy
+@inline ∂z_p(i, j, k, grid::RegularCartesianGrid, p) = δz_c2f(grid, p, i, j, k) / grid.Δz
```

Comment:
Hmmm, do we really need extra operators named specifically for pressure? Seems fine to me to call `δx_c2f(grid, p, i, j, k) / grid.Δx`, etc.

But maybe this point is moot as the operators will all be replaced by PR #283 at some point.

Reply:
Maybe just `∂x` etc. would be a better name as it's not particular to just pressure.

Reply:
The operator for pressure serves no functional purpose, as you point out --- it is purely introduced to enhance readability. We certainly can use `∂x_faa` (which is defined in `turbulence_closures/closure_operators.jl`). I think we should use functions as much as possible, so I'd prefer using a function here over hard-coding the derivative --- if this pattern is consistent with the finite volume implementation of these equations.

In `TurbulenceClosures`, I introduced a lot of such functions (see `∂x_u`, etc) in the hope that it would enhance the readability and clarity of the hyper-complicated functions involved in the turbulence closures.

I adopted the same philosophy here.

Some improvement could be had by defining `∂x_p` as an alias via

```julia
const ∂x_p = ∂x_faa
```

To avoid a proliferation of redundant functions (I think I did this in `TurbulenceClosures`, but forget when I was implementing it here).